js_modules/dagster-ui/packages/ui-core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -131,6 +136,50 @@ const SensorDryRun = ({repoAddress, name, currentCursor, onClose, jobName}: Prop
     setSubmitting(false);
   }, [sensorDryRun, sensorSelector, cursor, name]);
 
+  const onPersistCursorValue = useCallback(async () => {
+    const cursor = sensorExecutionData?.evaluationResult?.cursor;
+    if (!cursor) {
+      return;
+    }
+    const {data} = await setCursorMutation({
+      variables: {sensorSelector, cursor},
+    });
+    if (data?.setSensorCursor.__typename === 'Sensor') {
+      await showSharedToaster({message: 'Cursor value updated', intent: 'success'});
+    } else if (data?.setSensorCursor) {
+      const error = data.setSensorCursor;
+      await showSharedToaster({
+        intent: 'danger',
+        message: (
+          <Group direction="row" spacing={8}>
+            <div>Could not set cursor value.</div>
+            <ButtonLink
+              color={Colors.accentReversed()}
+              underline="always"
+              onClick={() => {
+                showCustomAlert({
+                  title: 'Python Error',
+                  body:
+                    error.__typename === 'PythonError' ? (
+                      <PythonErrorInfo error={error} />
+                    ) : (
+                      'Sensor not found'
+                    ),
+                });
+              }}
+            >
+              View error
+            </ButtonLink>
+          </Group>
+        ),
+      });
+    }
+  }, [sensorExecutionData?.evaluationResult?.cursor, sensorSelector, setCursorMutation]);
+
+  const onCommitTickResult = useCallback(() => {
+    onPersistCursorValue();
+  }, [onPersistCursorValue]);
```

Comment:
`onCommitTickResult` is just wrapping `onPersistCursorValue`? Why not just rename `onPersistCursorValue` to `onCommitTickResult`?